docker/etna-base/entrypoints/build.sh

```diff
@@ -40,6 +39,7 @@ if [ -z "$SKIP_RUBY_SETUP" ]; then
     mv /tmp/.git /app/vendor/bundle/*/ruby/*/bundler/gems/monoetna-*/
     cd /app
     bundle install
+    rm /app/vendor/bundle/*/bin/etna
```

Comment:
This prevents etna from being on the PATH of, say, the `polyphemus` bash environment since it's just confusing.  Still can run the etna bin from within the etna environment.